frontend/src/App.js

```diff
@@ -42,6 +46,29 @@ function App() {
             />
           </>
         )}
+        {hasRole(currentUser, "ROLE_USER") && (
+          <>
+            <Route
+              exact
+              path="/requests"
+              element={<RecommendationRequestIndexPage />}
+            />
+          </>
+        )}
+        {hasRole(currentUser, "ROLE_ADMIN") && (
```

Comment:
I think we want all three pages to be available to ROLE_USER.

And we probably need a separate page for the professor where they just have three buttons: [Accept],[Reject], [Mark Completed], [Return to Index Page].   Because those are the only things we are allowing a professor to do right now with a Recommendation Request, I think... and I think we have the endpoints we need to support those actions...